charts/hpe-flexvolume-driver/v3.0.0/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+appVersion: "3.0"
+description: A Helm chart for installing the HPE Volume Driver for Kubernetes FlexVolume plugin
+icon: https://raw.githubusercontent.com/hpe-storage/co-deployments/master/docs/assets/hpedev.png
```

Comment:
would prefer to add the icon to the parent directory and use it via icon: file://../logo.png for air-gapped users, thanks.

Reply:
done..thanks.